### PR TITLE
Update examples build to use kernel version v5.15

### PR DESCRIPTION
**Pull request by sanfern**
This PR to build examples with kernel v5.15

### CODE REVIEW

File: examples/tc/Makefile [hunk 1]
```diff
@@ -6,94 +6,214 @@ TOOLS_PATH := $(BPF_SAMPLES_PATH)/../../tools
...
+#ifeq ($(VMLINUX_H),)
+#	$(Q)$(BPFTOOL) btf dump file $(VMLINUX_BTF) format c > $@
+#else
+#	$(Q)cp "$(VMLINUX_H)" $@
+#endif
+
+#ifeq ($(VMLINUX_BTF),)
+#	$(error Cannot find a vmlinux for VMLINUX_BTF at any of "$(VMLINUX_BTF_PATHS)",\
+#		build the kernel or set VMLINUX_BTF variable)
+#endif
```
Comment (dthaler) [inline examples/tc/Makefile:170]: Remove commented out code

File: examples/xdp/Makefile [hunk 2]
```diff
@@ -1,98 +1,218 @@
...
+#ifeq ($(VMLINUX_H),)
+#	$(Q)$(BPFTOOL) btf dump file $(VMLINUX_BTF) format c > $@
+#else
+#	$(Q)cp "$(VMLINUX_H)" $@
+#endif
+
+#ifeq ($(VMLINUX_BTF),)
+#	$(error Cannot find a vmlinux for VMLINUX_BTF at any of "$(VMLINUX_BTF_PATHS)",\
+#		build the kernel or set VMLINUX_BTF variable)
+#endif
```
Comment (dthaler) [inline examples/xdp/Makefile:169]: Remove commented out code

**Review by dthaler — APPROVED** [review]

**Review by charleskbliu0 — APPROVED** [review]
Looks good

**Comment by sanfern** [comment]
This PR depends on https://github.com/l3af-project/l3afd/pull/245

**Pull request merged**